concent_api/core/message_handlers.py

```diff
@@ -832,7 +832,7 @@ def handle_send_force_payment(
             payment_ts              = parse_datetime_to_timestamp(claim_against_requestor.closure_time),
             task_owner_key          = requestor_ethereum_public_key,
             provider_eth_account    = provider_eth_address,
-            amount_paid             = claim_against_requestor.amount,
+            amount_paid             = int(claim_against_requestor.amount),
```

Comment:
Please add property to `DepositClaim` like `amount_as_int` which will do the converting and use it.

Reply:
Done